app/server/appsmith-plugins/restApiPlugin/src/main/java/com/external/helpers/DataUtils.java

```
@@ -150,26 +152,11 @@ public String parseFormData(List<Property> bodyFormData, Boolean encodeParamsTog
                         if (MultipartFormDataType.TEXT.equals(multipartFormDataType)) {
                             bodyBuilder.part(key, property.getValue());
                         } else if (MultipartFormDataType.FILE.equals(multipartFormDataType)) {
-
-                            MultipartFormDataDTO multipartFormDataDTO = null;
                             try {
-                                multipartFormDataDTO = objectMapper.readValue(
-                                        String.valueOf(property.getValue()),
-                                        MultipartFormDataDTO.class);
+                                populateFileTypeBodyBuilder(bodyBuilder, property, outputMessage);
                             } catch (JsonProcessingException e) {
                                 e.printStackTrace();
```

Hey sorry for this, I know this is not a part of this change, still since it is related I wanted to bring this up - the try catch here silently suppresses the exception from reading file data. Can we do something else here ? - like maybe return an `AppsmithExeception` or a warning message to the client that the file upload has failed ?